### PR TITLE
docs: added Integrate with Projects documentations

### LINKED ISSUE #201: How to integrate with other projects' docs
_TODO_

- [ ] Project Structure
- [ ] Versioned Docs

### PR DESCRIPTION
**Pull request by Ylikj**
Fixes: #201

### CODE REVIEW

**Comment by juzhiyuan** [comment]
cc @qier222 to review :)

**Comment by qier222** [comment]
The docs directory structure is incorrect

image
image

**Comment by juzhiyuan** [comment]
good catch, just updated.

**Comment by juzhiyuan** [comment]
image

Netlify is down due to unknown reasons 🤔 ignore it.

File: website/docs/general/integrate-with-project-docs.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+│   └── latest
+│       └── ...
+└── zh
+    └── latest
+        └── ...
+```
+
+## Configuration file
+
+This file is located in /docs/<locale>/latest/config.json, where locale is locale codes, please refer to https://www.science.co.il/language/Locale-codes.php, locale is all lowercase. Don't forget to remove the comment after copying JSON.
```
Comment (qier222) [inline website/docs/general/integrate-with-project-docs.md]: Why not wrap /docs/<locale>/latest/config.json with ``?

Comment (juzhiyuan) [reply website/docs/general/integrate-with-project-docs.md]: done

File: website/docs/general/integrate-with-project-docs.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+More information about sidebar can be found at https://v2.docusaurus.io/docs/next/sidebar
+
+```json
```
Comment (qier222) [inline website/docs/general/integrate-with-project-docs.md]: Maybe we should use jsonc instead of json since these codes have comments?

**Review by qier222 — CHANGES_REQUESTED** [review]
Looks good to me 👍

**Review by juzhiyuan — APPROVED** [review]

**Review by LiteSun — APPROVED** [review]

**Pull request merged**